src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.scss

```diff
@@ -3,3 +3,7 @@
 .tree-container {
   height: calc(100vh - vv.$tree-container-height);
 }
+
+.align-inline-btns {
+  margin-left: 15em;
+}
```

Comment:
Maybe we could accomplish this with boostrap classes as well.

Reply:
bootstrap [spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/) goes up to `ms-5` which is like 3 rem.